aviary/utils/process_input_decks.py

```diff
@@ -45,7 +45,9 @@ def create_vehicle(vehicle_deck=''):
     aircraft_values.set_val('mass_defect', val=10000, units='lbm')
     aircraft_values.set_val('problem_type', val=ProblemType.SIZING)
     aircraft_values.set_val(Aircraft.Electrical.HAS_HYBRID_SYSTEM, val=False)
-    aircraft_values.set_val(Aircraft.Design.RESERVES, val=4998)
+    aircraft_values.set_val(Aircraft.Design.RESERVES, val=4998, units='lbm')
+    aircraft_values.set_val(Aircraft.Design.RESERVES_FRACTION, val=0, units='unitless')
+    aircraft_values.set_val(Aircraft.Design.RESERVES_OPTION, val=0, units='unitless')
```

Comment:
Why is the reserves option being set to an invalid value?

Reply:
Aviary allows users to set the fuel in initial_guesses['reserves'] which overwrites the values in Aircraft.Design.RESERVES. So, I allow Aircraft.Design.RESERVES_OPTION to be 0 to indicate that we allow this behavior.